tools/nl/embeddings/input/base/sheets_svs.csv

```diff
@@ -2760,6 +2764,10 @@ Count_ThunderstormWindEvent,Number of thunderstorm wind events
 Count_TornadoEvent,Number of tornado events
 Count_TropicalStormEvent,Number of tropical storm events
 Count_UnemploymentInsuranceClaim_StateUnemploymentInsurance,Number of state unemployment insurance claims
+Count_Vehicle_InvolvedInCrash_InTransport,Number of vehicles crashed in transport 
+Count_VehicleCrashIncident_NationalHighway,Number of crash incidents occurred on the national highway

```

Comment:
"number of vehicle crash incidents..." same as below

Reply:
Since its counting the vehicles in-transport, shall we add sentence as "Number of vehicles in-transport involved in crash incidents"

Reply:
oh, i did not meant to include line 2767, that line looks good